### PR TITLE
feat: add replacement for deep-equal

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

File: docs/modules/deep-equal.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# deep-equal
+
+`deep-equal` has [roughly 50 dependencies](https://npmgraph.js.org/?q=deep-equal) while alternatives have zero.
```
Comment (43081j) [inline docs/modules/deep-equal.md:3]: i would reword this to be less specific about the number of dependencies i think (as it may change)

maybe just something like `deep-equal has various lighter weight and faster alternatives`

Comment (benmccann) [reply docs/modules/deep-equal.md]: thanks. good point. I've updated it

I also confirmed that saying faster is correct: https://www.npmjs.com/package/fast-deep-equal#performance-benchmark

**Review by 43081j — APPROVED** [review]

**Comment by benmccann** [comment]
Good thought. It's pretty good being zero dependencies. Though it is noticably larger than the other two on bundlephobia, so maybe I'll leave it off for now unless you still think we should add it

**Comment by 43081j** [comment]
no worries, works for me 👍
